crates/core/src/models/felt.rs

```diff
@@ -90,3 +90,106 @@ impl From<Felt252Wrapper> for U256 {
         U256::from_be_bytes(felt.to_bytes_be())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use hex::FromHex;
+
+    use super::*;
+
+    const MAX_ADDRESS: &str = "ffffffffffffffffffffffffffffffffffffffff";
+    const OVERFLOW_ADDRESS: &str = "010000000000000000000000000000000000000000";
+
+    const MAX_FELT: &str = "0800000000000011000000000000000000000000000000000000000000000000";
+    const OVERFLOW_FELT: &str = "0800000000000011000000000000000000000000000000000000000000000001";
```

Comment:
isn't there `FieldElement::MAX`, `H256::MAX`, etc or something to avoid defining these extra constants?

in any case we'd need fuzzer for these tests

Reply:
Changed to `FieldElement::MAX` where possible, unfortunately H60 for Address doesn't have the equivalent...